### DISCUSSION TITLE
Is there a way to hide checkbox in selectable mode for Tree View ?

### DISCUSSION THREAD

**Discussion by anviksuteiriy** [epicmaxco/vuestic-ui#4413] [Q&A]
I am implementing tree view in selectable mode and need to know if any children or parent checkbox could be hidden like we have disabled ?

**Comment by m0ksem** [comment]
You can do it with CSS:


https://ui.vuestic.dev/play#eNqdVFFv2jAQ/iuWn0AiZIyt2zJaaev60D1006j60kyViQ8wc5wodiiI5r/vbCclMGASfiG+O3/fd2f8beiXPO8vS6ARHRlIc8kMXMWKkNEDuy8AHgQ82y0hyyDNOMgomUPyB/hlTDVISAzwO4zrmPqySNkdZt1vE/WlbCLB72GVM8UDJqXdh8joOHOSSKbt6dQEFzF1SggZ10TEYUZks6kBa+7+IhOqE9NeTLvk5YVYcgUxJVXlYMMcgUZhqz/carOW9rO/ZIHBTgMLHgRcaCuTk914kikDyjw9CUTBHk3nRD4IJLBpl2wsOwIi6zpC8Qo+xwoljcKG3OpICpEb/MaZZIUhHKaslMYfViyFCPv55oO2RTcpzgyLSKdLLq9Ix1U2M65HEpHH31hr4/XQHv2OeGC/BI/IoC6zS7IJ3jDyXeOYZlmxdoRNNpkLyQtQLaw9vAbzbevUDu64nCSHoO1qJh8RU5SwlzzC7fgd5bC3ZbnFO7C3v4fRlL77f2kzO792swf6fX9Ov62WvK6LA7qOC9lmttG9q/1wztV6LR9PajnK+Ol8xsGb8ygHB//AHuCf0x6v6uJPhW8RH2Lz+miPitQ+wCBleX+hM4WW6KjiOoFmhtbjcWKKnqmNSIJS2HBM58bkOgrDUlg7tak+h2XYlGWlCVMmFCJ7T0QbqJDTaHSOqZjtMSZZmgsJxY/ciEztMqNpZs/fXaz1UPCMNeYD8YVeeYk/C9BQLNEYX3OGFTNAW7Hpm/EdrLzF+CT6fYmGfSr5C3QmS6vRl30tFUfZrTqn9tYNUKjZvb5ZoU/qpikr1E3D1bupXp9ofSt32B++TrH6C7z0AJM=
